rust/tw_any_coin/tests/chains/ton/ton_sign.rs

```diff
@@ -15,7 +15,7 @@ use tw_proto::TheOpenNetwork::Proto::mod_Transfer::OneOfpayload as PayloadType;
 /// The same Cell can be BoC encoded differently.
 /// Use this function to compare inner Cells closing eyes on the encoding.
 #[track_caller]
-fn assert_eq_boc(left: &str, right: &str) {
+pub fn assert_eq_boc(left: &str, right: &str) {
```

Comment:
Why public?

Reply:
I have changed it to `pub(crate)` as ton_sign_wallet_v5r1 also use it.